Automattic/wp-calypso
# Update checkout with ecommerce

#### Changes proposed in this Pull Request

* Updates `my-stes/checkout` with ecommerce constants etc

#### Testing instructions

* To review, I'd recommend going through the commits individually to make it easier
* Test by going through checkout with business plan and make sure it still behaves as expected
